k8s/deployment.yaml

```
@@ -0,0 +1,22 @@
+    metadata:
+      labels:
+        app: hot-deal-dev
+    spec:
+      containers:
+        - name: hot-deal-dev
+          image: yeseulhong/hot-deal-dev:latest
+          ports:
+            - containerPort: 8080
+          envFrom:
+            - secretRef:
+                name: jwt-secret
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)